Aglis - Psyche Bomb Trial Hub
Meru
Skip hiding model parts
--- original
+++ modified
@@ -307,6 +307,7 @@
 mov 0x700, stor[24]
 mov 0x5, stor[25]
 gosub inl[:LABEL_3]
+jmp inl[:THE_DISMEMBERMENT_PLAN]
 call 288, 0x0
 call 288, 0x2
 call 288, 0x3
@@ -323,6 +324,7 @@
 call 288, 0x15
 call 288, 0x16
 call 288, 0x17
+THE_DISMEMBERMENT_PLAN:
 call 97, 0x5
 gosub inl[:LABEL_0]
 call 97, 0x0
@@ -342,12 +344,6 @@
 call 97, 0x8
 gosub inl[:LABEL_0]
 call 97, 0x0
-call 288, 0x4
-call 288, 0x5
-call 288, 0xa
-call 288, 0xb
-call 288, 0x18
-call 288, 0x19
 mov 0x3, stor[24]
 gosub inl[:LABEL_2]
 mov 0x1, stor[24]
